apps/win32/wdio.conf.js

```diff
@@ -165,17 +152,15 @@ exports.config = {
   /**
    * Function to be executed after a test (in Mocha/Jasmine).
    */
-  afterTest: function (test) {
-    // if test passed, ignore, else take and save screenshot.
-    /* UPDATE: I want to take screenshots after every test to help gauge certain CI failures.
-     * I will re-enable this once I am able to determine what's causing some CI failures
-     */
-    // if (test.passed) {
-    //   return;
-    // }
+  afterTest: function (test, context, results) {
+    // if test passed, ignore, else take and save screenshot. Unless it's the first test that boots the app,
```

Comment:
Thinking in a longer view; could it later be worth always taking a screenshot, and providing an easy mechanism for comparing before/after changes?